### ISSUE TITLE
Use src/setup.py for the installation mode --disable-editable --disable-wheels

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage#34633]
<div id="comment:0"></div>

... instead of using `pkgs/sagemath-standard/setup.py` (which can then be transformed to the modularized build - #34587).

Depends on #32874

Component: **build**

Work Issues: **Cleaner?**

Branch/Commit: **[u/mkoeppe/use_src_setup_py_for_the_installation_mode___disable_editable___disable_wheels](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/use_src_setup_py_for_the_installation_mode___disable_editable___disable_wheels) @ [`8a3b586`](https://github.com/sagemath/sagetrac-mirror/commit/8a3b5862939deee99671dba9b0227c6a70dcf379)**

_Issue created by migration from https://trac.sagemath.org/ticket/34633_

**Comment by mkoeppe** [comment]
Dependencies: **#32874**

**Comment by mkoeppe** [comment]
Branch: **[u/mkoeppe/use_src_setup_py_for_the_installation_mode___disable_editable___disable_wheels](https://github.com/sagemath/sagetrac-mirror/tree/u/mkoeppe/use_src_setup_py_for_the_installation_mode___disable_editable___disable_wheels)**

**Comment by mkoeppe** [comment]
Commit: **[`8a3b586`](https://github.com/sagemath/sagetrac-mirror/commit/8a3b5862939deee99671dba9b0227c6a70dcf379)**

**Comment by mkoeppe** [comment]
Work Issues: **Cleaner?**

**Comment by mkoeppe** [comment]
<div id="comment:3"></div>

Last 10 new commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/a683934893dbed93451c6ef3b58722bb14f4c850"><code>a683934</code></a></td><td><code>configure.ac (--enable-wheels): New</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/f2d1576d13e1bd6299466172eecb9baecfdd9723"><code>f2d1576</code></a></td><td><code>configure.ac: Improve docstring</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/05b44d76ecee64468ca1ee79d5b0559fc28c26d5"><code>05b44d7</code></a></td><td><code>build/pkgs/{sage_conf,sage_docbuild,sage_setup,sage_sws2rst}/spkg-install: Handle SAGE_WHEELS</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/dde4bdb9c4553d9c25246a45d98a784e1c1cd8b3"><code>dde4bdb</code></a></td><td><code>src/sage/misc/package_dir.py: Fix docstring markup</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/ffe67e8c6ad0efd6481aabec24dd0a4a1d603426"><code>ffe67e8</code></a></td><td><code>is_package_or_sage_namespace_package_dir: Add option distribution_filter</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/11507a5f7e598035b4be6c00e89919190b271dc4"><code>11507a5</code></a></td><td><code>build/pkgs/sagelib/spkg-install: No need to uninstall before installing - cleaner is back in</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/974c7fd8aa4dce5b2376ad430d1e0769d91b7912"><code>974c7fd</code></a></td><td><code>src/MANIFEST.in: Remove redundant entries for MANIFEST.in, pyproject.toml</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/25898ba992588a8dc2df07e40c12a4dcee9909f3"><code>25898ba</code></a></td><td><code>src/MANIFEST.in: Exclude generated file farey_symbol.h; add comments</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/368f5055028e964bfc6b62f2b850c822256aa2a9"><code>368f505</code></a></td><td><code>Merge #32874</code></td></tr><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/8a3b5862939deee99671dba9b0227c6a70dcf379"><code>8a3b586</code></a></td><td><code>Use src/setup.py for the installation mode --disable-editable --disable-wheels</code></td></tr></table>

**Comment by mkoeppe** [comment]
Closing in favor of:
- #37973
